scaffolder-templates/gitlab-workflows/advanced-workflow/template.yaml

```diff
@@ -111,7 +111,7 @@ spec:
                 gitlabHost:
                   title: Gitlab Instance Name
                   type: string
-                  default: gitlab.cee.redhat.com
+                  default: GITLAB_HOST
```

Comment:
this makes the impression it should be an env-var.
perhaps just in lower case: `gitlab-host` ?